wso2am/Dockerfile

```diff
@@ -16,7 +16,7 @@
 
 # ------------------------------------------------------------------------
 
-FROM ubuntu:latest
+FROM frolvlad/alpine-oraclejdk8:latest
```

Comment:
@chintana We might need not be able to this base Docker image due to Oracle JDK licensing issues. Shall we change this to alpine:latest and setup JDK separately?